cx-sig-proposed-README.md

```diff
@@ -0,0 +1,29 @@
+# Spinnaker Contributor Experience Special Interest Group
+
+## Meeting Info
+* Weekly on Thursdays at 10 AM US/Pacific ([See in your time zone](https://www.thetimezoneconverter.com/?t=10am&tz=San%20Francisco))
+* [Agenda and notes](https://hackmd.io/@spinnaker-cx/ry03Vk6LP)
+* [Meeting link]
+
+## Roles & Responsibilities of the SIG Leads
+
+### SIG Lead(s):
+
+**[Ige Adetokunbo Temitayo](https://github.com/ExitoLabs)**  
+DevOps and Site Reliability Engineer at Emirates Group
+
+**[Nikema Prophet](https://github.com/prophen)**  
+Associate Community Marketing Manager at Armory
+
+### SIG Lead Responsibility
+
+* Coordinate communication.
```

Comment:
Clarity needed: coordinate communication of what?

Reply:
1 idea I see here is the frequency by which a SIG updates the larger community. Think of a community meeting where SIG are chosen to give an update to the community, that is where this will happen/

Reply:
Adding clarification here is a good idea, and nice example from @markyjackson-taulia

To me this is a pretty general designation that a SIG Lead's responsibility, among other things, is to coordinate communication with SIG members and community stakeholders in the SIG's interest area about the SIG's activities (meetings, initiatives, decisions, etc.)

Reply:
```suggestion
* Coordinate communication with SIG participants and the larger community.
```